doc/maintaining/datastore.rst

```diff
@@ -411,6 +411,36 @@ You can find more information about the formatting of dates in the `date/time ty
 
 .. _date/time types section of the PostgreSQL documentation: http://www.postgresql.org/docs/9.1/static/datatype-datetime.html
 
+.. _filters:
+
+Filters
+-------
+
+Filters define the matching conditions to select from the DataStore. A filter is defined as follows::
+
+    {
+        "resource_id":  # the resource ID (required)
+        "filters": {
+            # column name: # field value
+            ...:  # other user-defined filters
+	}
+    }
+
+Filters must be supplied as a dictonary. Filters are used as basic `WHERE =` statements.
+The filters have to be valid key/value pairs. The key must be a valid column name
+and the value must match the respective column type.
+See :ref:`valid-types` for details on which types are valid.
+
+Example::
+
+    {
+        "resource_id":  "5f38da22-7d55-4312-81ce-17f1a9e84788",
+        "filters": {
+            "name": "Fred",
+            "dob":  "1994-7-07"
+	}
+    }
+
```

Comment:
Thank you for adding this section. Filters also support passing a list of strings as values that are treated as "column must match one of these values", could you add a simple example for that as well?

Reply:
@wardi new wording and example for Lists has been added.